cli/tests/activate.bats

```diff
@@ -2487,6 +2487,58 @@ EOF
   assert_success
 }
 
+@test "profile: JUPYTER_PATH not modified when Jupyter is not installed" {
+  project_setup
+
+  # Shouldn't be set by default.
+  run "$FLOX_BIN" activate -- \
+    echo JUPYTER_PATH is '$JUPYTER_PATH'
+  assert_success
+  assert_output "JUPYTER_PATH is"
+
+  # Should respect existing variable from outside activation.
+  JUPYTER_PATH="/fakepath" run "$FLOX_BIN" activate -- \
+    echo JUPYTER_PATH is '$JUPYTER_PATH'
+  assert_success
+  assert_output "JUPYTER_PATH is /fakepath"
+}
+
+@test "profile: JUPYTER_PATH is modified when Jupyter is installed" {
+  # Mock contains both extensions but only one is used in each install.
+  export _FLOX_USE_CATALOG_MOCK="$GENERATED_DATA/resolve/jupyter_with_extensions.json"
```

Comment:
Nit: I prefer these two be different tests with the common setup extracted into a helper function.

Reply:
That would normally be my preference too but I think it makes these tests either slower to run (if we duplicate the setup and assertions) or harder to maintain (if we cut down on the assertions or make them global vars).